cds_ils/migrator/json_record_loader.py

```diff
@@ -65,7 +65,7 @@ def create_record(
                 )
                 legacy_pid_type = cls.get_legacy_pid_type_by_provider(provider)
 
-                if legacy_pid_type:
+                if legacy_pid_type and legacy_id_key == "legacy_recid":
```

Comment:
if we integrate this change we will not mint for these types anymore
```
CDS_ILS_BORROWING_REQ_LEGACY_PID_TYPE = "lillb"
CDS_ILS_ACQ_ORDER_LEGACY_PID_TYPE = "lacqid"
CDS_ILS_ITEM_LEGACY_PID_TYPE = "litmid"
```
also legacy recid type is missing for series which causes it to mint legacy pid of **document** type for **series** which causes further problems in migration